contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
+
+            // Add last question and answer
+            const unusedLastAnswers = lastQuestion.answers.filter(answer => !answersUsed.includes(answer.id))
+
+            if (unusedLastAnswers.length) {
+                newPath.unshift({ question: lastQuestion, answer: unusedLastAnswers[0] })
+                answersUsed.push(unusedLastAnswers[0].id)
+                neededNextQuestions.unshift(lastQuestion.id);
+            } else {
+                newPath.unshift({ question: lastQuestion, answer: lastQuestion.answers[0] })
+            }
+
+            // Loop through remaining questions
+            for (let i = this.questions.length - 2; i >= 0; i--) {
+                const currentQuestion = this.questions[i]
+                if (neededNextQuestions.length) {
+                    const validAnswers = currentQuestion.answers.filter(answer => answer.nextQuestion && answer.nextQuestion.sys.id == neededNextQuestions[0])
+                    const validUnusedAnswers = validAnswers.filter(answer => !answersUsed.includes(answer.id))
+
+                    currentAnswer = validUnusedAnswers.length ? validUnusedAnswers[0] : validAnswers[0]
+
+                    if (validUnusedAnswers.length) {
+                        answersUsed.push(currentAnswer.id)
+                    }
+
+                    neededNextQuestions.unshift(currentQuestion.id)
+
+                } else {
+                    const unusedAnswers = currentQuestion.answers.filter(answer => !answersUsed.includes(answer.id))
+
+                    currentAnswer = unusedAnswers.length ? unusedAnswers[0] : currentQuestion.answers[0]
+
+                    if (unusedAnswers.length) {
+                        neededNextQuestions.unshift(currentQuestion.id)
+                        answersUsed.push(currentAnswer.id)
+                    }
+                }
+
+                newPath.unshift({ question: currentQuestion, answer: currentAnswer })
+            }
+
+            // Check through new path and remove later questions if nextQuestion is undefined (ie path contains early answer that shortens the user journey)
+            for (let i = 0; i < newPath.length; i++) {
+                if (newPath[i].answer && !newPath[i].answer.nextQuestion) {
+                    newPath.splice(i + 1, newPath.length - i + 1)
+                    break;
+                } else if (newPath[i].answer && newPath[i].answer.nextQuestion.sys.id !== newPath[i + 1].question.id) {
+                    const nextQuestion = newPath.findIndex((question) => question.question.id === newPath[i].answer.nextQuestion.sys.id, i + 1)
+                    newPath.splice(i + 1, nextQuestion - (i + 1))
+                    continue;
+                }
+            }
+            allAnswerPaths.push(newPath)
+        }
+        this.pathsForAllPossibleAnswers = allAnswerPaths.map((path) => {
+            const userJourney = new UserJourney(path, this);
+            userJourney.setRecommendation(this.recommendation, path);
```

Comment:
Are there some changes missing from the PR? `UserJourney` doesn't appear to have any changes in it, and in the current version (that I can see) `setRecommendation` only takes one argument (the recommendation).

Reply:
So I think this is from me playing around with getting the chunks on the UserJourney, which we already have some scaffolding for but we don't seem to have used. I went back and forth and eventually decided to just leave things as is but left the extra param in by mistake. I still can't decide if we should have the chunks mapped onto the UserJourney, thoughts?